server/odc-server/src/main/java/com/oceanbase/odc/server/OdcServer.java

```diff
@@ -70,6 +72,10 @@ public OdcServer(@Qualifier("metadbMigrate") AbstractMetaDBMigrate metadbMigrate
      * @param args
      */
     public static void main(String[] args) {
+        if (Objects.equals(SystemUtils.getEnvOrProperty("ODC_SERVER_MODE"), "TASK_EXECUTOR")) {
```

Comment:
ODC_SERVER_MODE and TASK_EXECUTOR  put in JobEvnConstants

Reply:
Okay, Done!